.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Dataflow_Java11.groovy

```diff
@@ -19,9 +19,11 @@
 import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 
+final String JAVA_11_HOME = '/usr/lib/jvm/java-11-openjdk-amd64'
+final String JAVA_8_HOME = '/usr/lib/jvm/java-8-openjdk-amd64'
```

Comment:
These should be loaded from env or be parameterized.
Also these values should work for windows as well.

Reply:
Good point - I agree.  This was copied from some other job ([VR_Direct_Java11](https://github.com/apache/beam/blob/fa37fc5e176e72fd346b9a0bb907d9726b33d018/.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Direct_Java11.groovy)). 

If I'm not mistaken, adding new env variables to Jenkins has to go through asf infra so that they add the variables and then we can easily use them. Am I right @Ardagan ?

Reply:
Created a separate issue for this: https://issues.apache.org/jira/browse/BEAM-8919